metadata.go

```diff
@@ -947,11 +947,11 @@ func getColumnMetadata(session *Session, keyspaceName string) ([]ColumnMetadata,
 	return columns, nil
 }
 
-func getTypeInfo(t string, logger StdLogger) TypeInfo {
+func getTypeInfo(t string, protoVer byte, logger StdLogger) TypeInfo {
 	if strings.HasPrefix(t, apacheCassandraTypePrefix) {
 		t = apacheToCassandraType(t)
```

Comment:
we can get rid of `apacheToCassandraType` function now right? And just call `getCassandraLongType`?

Reply:
Indeed, as always great review. Updated.